pkg/client/http/mapper.go

```diff
@@ -42,35 +42,30 @@ var mappers = map[string]client.ParamMapper{
 	constant.QueryStrings: queryStringsMapper{},
 	constant.Headers:      headerMapper{},
 	constant.RequestBody:  bodyMapper{},
+	constant.RequestURI:   uriMapper{},
 }
 
 type requestParams struct {
-	Header http.Header
-	Query  url.Values
-	Body   io.ReadCloser
+	Header    http.Header
+	Query     url.Values
+	Body      io.ReadCloser
+	URIParams url.Values
 }
 
 func newRequestParams() *requestParams {
 	return &requestParams{
-		Header: http.Header{},
-		Query:  url.Values{},
-		Body:   ioutil.NopCloser(bytes.NewReader([]byte(""))),
+		Header:    http.Header{},
+		Query:     url.Values{},
+		Body:      ioutil.NopCloser(bytes.NewReader([]byte(""))),
+		URIParams: url.Values{},
 	}
 }
 
 type queryStringsMapper struct{}
 
 // nolint
 func (qs queryStringsMapper) Map(mp config.MappingParam, c *client.Request, rawTarget interface{}, option client.IOption) error {
```

Comment:
Can "qs" is defined as pointer?

Reply:
because it will not use anything within the queryStringsMapper struct, and it is a lite struct. So I think it might be better to make it a non-pointer function